tee-worker/litentry/core/assertion-build/src/achainable/mod.rs

```diff
@@ -111,7 +111,7 @@ pub fn request_achainable(
 		)
 	})?;
 
-	Ok(false)
+	Ok(result)
```

Comment:
🙃

Reply:
Well spot, and also I found that actually this was already like this before the AbortStrategy PR: https://github.com/litentry/litentry-parachain/pull/2707/files#diff-c23f6a3018245418b1a8d1c1dc7187aeae2c92785b32294c437e46849be35aaeR114, i.e. the original implementation might have problems since day 1.

Reply:
I see - thanks.

I'm wondering if it ever worked before - including our data provider test. cc @0xverin @BillyWooo

Reply:
@Kailai-Wang @higherordertech 
I think[ it worked before](https://github.com/litentry/litentry-parachain/pull/2707/files#diff-c23f6a3018245418b1a8d1c1dc7187aeae2c92785b32294c437e46849be35aaeL98) .